modules/rosa-adding-instance-types.adoc

```diff
@@ -0,0 +1,50 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-managing-worker-nodes.adoc
+
+[id="rosa-adding-instance-types_{context}"]
+= Adding instance types
+
+[role="_abstract"]
+After a machine pool is created, the instance type cannot be changed. To add a different instance type for worker nodes, you must create a new machine pool for the additional instance type.
+
+.Procedure
+
+. To add an instance type with a new machine pool, enter the following command:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=<cluster-name> <machinepool_ID> --instance-type=<type>
+----
++
+[NOTE]
+====
+Some of the supported AWS compute instance types are: `M5.xlarge`, `M5.2xlarge`, and `M5.4xlarge`. For a complete list of supported instance types, see the {product-title} Service Definition.
+====
++
+.Example
```

Comment:
s/.Example/.Example output
From our doc_guidelines: 
> prepend the code block for the output with the title .Example output

Reply:
This is not output, it is a standalone example of a command input. ISG supports this on page 148. But I did move the note to follow the example, which I think makes it more obvious that it's an example of a command input. That's probably what made it confusing.

Reply:
Our convention is to use the example header right above the code block as you do later in line 44.